%YAML:1.0
cameraMatrix: !!opencv-matrix
   rows: 3
   cols: 3
   dt: d
   data: [ 1.1929619385153785e+03, 0., 6.3975684508994709e+02, 0.,
       1.1885379520161100e+03, 3.5834558023450467e+02, 0., 0., 1. ]
distCoeffs: !!opencv-matrix
   rows: 1
   cols: 5
   dt: d
   data: [ -4.7021633393887619e-01, 1.8783433335687665e+01,
       -7.0807159947554365e-03, 2.8456522520787174e-03,
       -2.8794239860489142e+01 ]
camera_transform:
   position:
      x: 1.3993772132348214e-01
      y: 5.3047395205734171e-01
      z: -4.7938340766340154e-01
   orientation:
      x: 0.
      y: 0.
      z: 0.
      w: 1.
